src/neorados/cls/fifo.h

```diff
@@ -1359,6 +1423,98 @@ class FIFO {
 	return;
       }
     }
+
+    void list2() {
```

Comment:
after separated into its own client this will just be called `list` and replace the above call